### DISCUSSION TITLE
Keyboard

### DISCUSSION THREAD

**Discussion by bdbch** [tailwindlabs/heroicons#525] [Ideas]
Maybe I'm blind but I can't find a keyboard icon right now.
Would be a great addition to heroicons!

**Comment by dalkommatt** [comment]
+1 keyboard icon

**Comment by ravanscafi** [comment]
duplicated of: https://github.com/tailwindlabs/heroicons/discussions/418

anyway, a keyboard icon (to create a button to `show keyboard shortcuts` for example) would be much appreciated! ❤️
